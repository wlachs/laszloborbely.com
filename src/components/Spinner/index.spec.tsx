import { cleanup, render } from '@testing-library/react';
import { afterEach, describe, expect, it } from 'vitest';

import { Spinner } from './';

describe('Spinner component', () => {
	afterEach(() => {
		cleanup();
	});

	it('should render spinner', () => {
		const { container } = render(<Spinner />);
		expect(container).toMatchSnapshot();
	});

	it('should align center', () => {
		const { container } = render(<Spinner />);
		expect(container.children[0].className).toContain(
			'flex justify-center',
		);
	});
});
