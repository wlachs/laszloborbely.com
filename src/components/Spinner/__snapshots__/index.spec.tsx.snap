// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Spinner component > should render spinner 1`] = `
<div>
  <div
    class="d-flex justify-content-center"
  >
    <span
      class="spinner-border primary-text"
      role="status"
    />
  </div>
</div>
`;
